src/server/scripts/Outland/CoilfangReservoir/TheSlavePens/boss_ahune.cpp

```diff
@@ -169,19 +168,14 @@ class boss_ahune : public CreatureScript
     {
         boss_ahuneAI(Creature* creature) : BossAI(creature, DATA_AHUNE)
         {
-            Initialize();
-        }
-
-        void Initialize()
-        {
-            me->SetFlag(UNIT_FIELD_FLAGS, UNIT_FLAG_REMOVE_CLIENT_CONTROL);
+            me->SetControlled(true, UNIT_STATE_ROOT);
```

Comment:
are you sure about this? really it should be only in ctor? shouldn't the NPC be rooted even after wipe for example? or do I miss something?